aws/launch_template.go

```diff
@@ -0,0 +1,87 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/report"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+// Returns a formatted string of Launch Template Names
+func getAllLaunchTemplates(session *session.Session, region string, excludeAfter time.Time, configObj config.Config) ([]*string, error) {
```

Comment:
`region` variable seems to be unused

Reply:
Good catch @denis256, thank you. This has now been fixed.

I realized that I made this error by copying over an implementation from the `launch-config.go` file. A quick scan of the repo with "region string" shows a bunch more functions that this same issue. I will file an issue for them.

Reply:
#404